src/v4l.c

```diff
@@ -41,11 +41,18 @@ so, delete this exception statement from your version.
 #include "keyboard.h"
 #include "allowed_input_t.h"
 
-#if HAVE_LINUX_VIDEODEV_H
+#if HAVE_LIBV4L1_VIDEODEV_H || HAVE_LINUX_VIDEODEV_H
```

Comment:
TBH, those checks look quite convoluted now - you're free to throw the current status quo away and do a new implementation that checks for v4l1 and v4l2, respectively.

Reply:
Thanks - that would definitely make it possible to simplify the code.  Since v4l1 is no longer supported by kernel 2.6.38 and later, can we ditch it completely and support only v4l2?

Reply:
As the project is more or less in maintenance mode, please try to make a change that as noninvasive as possible. I guess cleaning up the header check would suffice.

Reply:
Thanks.  I'll leave the PR as-is in that case.  It would take a moderately invasive change to add the feature that I actually need, so if this project is maintenance-only then I'll implement what I need as a standalone codebase rather than as part of `x11vnc`.

Feel free to merge or discard this PR as you see fit.